homeassistant/components/tuya/binary_sensor.py

```diff
@@ -256,6 +256,23 @@ class TuyaBinarySensorEntityDescription(BinarySensorEntityDescription):
         ),
         TAMPER_BINARY_SENSOR,
     ),
+    # Smart Water Timer
+    "sfkzq": (
```

Comment:
Can you provide links to documentation?

Reply:
Unfortunately there is no documentation links for this device, as far as I could search for.

Reply:
https://developer.tuya.com/en/docs/iot/Zigbee-Water-valve-controller-access-standard?id=Kbahvojbqzvgl perhaps?